Developer.bundle

```diff
@@ -22,16 +22,12 @@ brew 'brew-cask-completion'
 brew 'bundler-completion'
 brew 'gem-completion'
 brew 'open-completion'
-brew 'vagrant-completion'
 
 # Latest versions of essential tools
 brew 'wget'
 brew 'curl'
 brew 'git'
 
-# PHP and tools
-brew 'php71'
```

Comment:
How would PHP be installed on the host? It's very useful for running Composer, other commands and even some unit tests on the host quickly. We could install PHP 7.3 now locally.

Reply:
Hi @hkdobrev Catalina comes with php 7.3 pre-installed.

Reply:
@hkdobrev sure.

Reply:
type php --version on preinstalled Catalina and you will see PHP 7.3.11 (cli)
Catalina comes with php 7.3 pre-installed....
@hkdobrev The latest macOS 10.15 Catalina comes with Apache 2.4 pre-installed....
<img width="504" alt="Screenshot 2020-06-30 at 1 22 06" src="https://user-images.githubusercontent.com/47556121/86061878-3f572280-ba70-11ea-9e41-4ef8a976945f.png">